repair/row_level.cc

```diff
@@ -561,6 +562,7 @@ class repair_writer {
 
     future<> write_end_of_stream(unsigned node_idx) {
         if (_mq[node_idx]) {
+          return with_semaphore(_sem, 1, [this, node_idx] {
```

Comment:
This looks drastic, it is done for every partition.

Reply:
Is there no simpler solution?

Reply:
Scratch that, it's only at the end.

Reply:
Yes, it is called at the end of the stream only.